.github/workflows/workflow.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish NPM package
+on:
+  release:
+    types: [created]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm cit
+  publish:
+    runs-on: ubuntu-latest
+    needs: test
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm i
```

Comment:
@neelkamath Minor change. I am using `yarn` so you need to replace this. Thank you